src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -77,8 +77,8 @@ public sealed partial class Font : MarshalByRefObject, ICloneable, IDisposable,
         /// Gets the face name of this <see cref='Font'/> .
         /// </summary>
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-#if !NETCORE
         [Editor ("System.Drawing.Design.FontNameEditor, " + Consts.AssemblySystem_Drawing_Design, typeof (System.Drawing.Design.UITypeEditor))]
```

Comment:
This constant is not present in the corefx codebase (presumably it is in the Mono codebase).   Either define a constant here, or just inline the string.  Also you cannot use `typeof(UITypeEditor)` here, you need to use the assembly qualified name since UITypeEditor lives upstack.

This should change to (effectively):
```C#
[Editor("System.Drawing.Design.FontNameEditor, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", "System.Drawing.Design.UITypeEditor, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"]
```